fbw-common/src/systems/instruments/src/ND/FmMessages.tsx

```diff
@@ -37,7 +37,7 @@ export class FmMessages extends DisplayComponent<FmMessagesProps> {
   );
 
   private readonly visible = this.props.mode.map((mode) => {
-    if (mode === EfisNdMode.ROSE_ILS || mode === EfisNdMode.ROSE_VOR) {
+    if (mode === EfisNdMode.ROSE_ILS || mode === EfisNdMode.ROSE_VOR || this.activeMessages.length === 0) {
```

Comment:
Bug. The MappedSubject is not subscribing to this.activeMessages.

Reply:
Indeed, thanks!